lib/components/CursorTooltip/CursorTooltip.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useRef } from 'react'
+import { Tooltip } from '@mui/material'
+import clsx from 'clsx'
+import { EMPTY_STRING } from '../../consts'
+
+import '../Tooltip/tooltip.scss'
+
+export type CursorTooltipProps = {
+  children: React.ReactElement
+  clear?: boolean
+  data?: React.ReactElement | string
+  extraClass?: string
+  extraPopperClass?: string
+  showDelay?: number
+  disappearAfterTimeout?: number
+} & React.ComponentProps<typeof Tooltip>
+
+function CursorTooltip({
+  children,
+  clear = false,
+  data = EMPTY_STRING,
+  extraClass = EMPTY_STRING,
+  extraPopperClass = EMPTY_STRING,
+  showDelay = 1000,
+  disappearAfterTimeout = Infinity,
```

Comment:
btw, don't `leaveDelay` (or/and maybe `leaveTouchDelay`) / `enterDelay` of MUI Tooltip serve these purposes?

Reply:
Yes, actually it can simplify the code.
Since we still want the functionality of showing again if moving and stopping over the container, we should still use that component, otherwise we can actually use toolTip with those props. 
Something like this
```
<Tooltip
      enterDelay={showDelay}
      leaveDelay={SOME_NUMBER_IF_NEEDED}
      enterTouchDelay={1000}
      leaveTouchDelay={SOME_NUMBER_IF_NEEDED}
      classes={{
        tooltip: classes,
        arrow: 'tooltip-arrow',
        popper: extraPopperClass
      }}
      placement='top'
      arrow
      followCursor
      {...rest}
      title={data}
    >

Reply:
let's still have the CursorTooltip